<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGameClientClass</class>
 <widget class="QMainWindow" name="QtGameClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtGameClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>731</width>
      <height>125</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="3" column="1">
      <widget class="QLineEdit" name="targetIp"/>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="connectBtn">
       <property name="text">
        <string>Connect!</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="connectionLbl">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Connect to A Game</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ipAddrlbl">
       <property name="text">
        <string>Ip Addr</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="playerName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="playerNameLbl">
       <property name="text">
        <string>Nickname</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGameClient.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>connectBtn</sender>
   <signal>clicked()</signal>
   <receiver>QtGameClientClass</receiver>
   <slot>connectToGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connectToGame()</slot>
 </slots>
</ui>
